NGettext.Wpf/EnumTranslation/EnumMsgIdAttribute.cs

```diff
@@ -6,6 +6,10 @@ public class EnumMsgIdAttribute : Attribute
     {
         public EnumMsgIdAttribute(string msgId)
         {
+            if (msgId.Contains("|"))
+            {
+                msgId = msgId.Substring(msgId.IndexOf("|") + 1);
```

Comment:
We need the context too.

If there is a context, that should be used with Catalog.GetParticularString()